atom/browser/api/atom_api_session.cc

```diff
@@ -416,6 +418,12 @@ void Session::DoCacheAction(const net::CompletionCallback& callback) {
                  callback));
 }
 
+void Session::ClearHSTSData() {
+  auto request_context = request_context_getter_->GetURLRequestContext();
+  request_context->transport_security_state()->
```

Comment:
`storage_partition->GetNetworkContext()->ClearNetworkingHistorySince` - see browsing_data_remover_impl.cc

Reply:
also you can't access `request_context_getter_` on the UI thread. The other methods in `Session` that use `request_context_getter_` post to the IO thread. The storage partition can be accessed on the UI thread -  see `ClearStorageData`